Builder/inc/CodeBuilder.hpp

```diff
@@ -0,0 +1,32 @@
+#ifndef CODEBUILDER_H
+#define CODEBUILDER_H
+
+#include <string>
+#include <ostream>
+#include <vector>
+
+using namespace std;
+
+struct Field
+{
+    string name{}, type{};
+
+    Field(const string& name, const string& type);
+    string str();
+};
+
+class CodeBuilder
```

Comment:
The scope of the builder design pattern is to hide the complex logic related to the creation of an object. It would be better to create a class / struct called Class and have a Class member variable which shall be built in the CodeBuilder class.

Reply:
Fixed this.